databrowser/src/components/pill/PillGroup.vue

```diff
@@ -60,18 +40,13 @@ export default defineComponent({
       }
     );
 
-    function isSelected(current: string) {
-      return selected.value == current;
-    }
-
     function changeSelectedItem(item: string) {
       selected.value = item;
       context.emit('selectedChange', selected.value);
```

Comment:
Also here I think it could be dangerous to use the value to return the selection. I would use the index, because it is more reliable.

Reply:
I would switch the selection state completely to the index and not to the value.